src/document.h

```diff
@@ -6,28 +6,26 @@ struct UndoItem {
     Vector<Selection> path, selpath;
     Selection sel;
     unique_ptr<Cell> clone;
-    size_t estimated_size;
+    size_t estimated_size{ 0 };

```

Comment:
I would just have done `= 0` like we already have in some places. If there's a reason `{}` is better, I'd put a space ahead of the first `{`.